src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3321,6 +3342,20 @@ void ViewProviderSketch::onCameraChanged(SoCamera* cam)
         Base::Interpreter().runStringObject(cmdStr.toLatin1());
     }
 
+    Gui::View3DInventor* view = qobject_cast<Gui::View3DInventor*>(this->getActiveView());
+    if (view) {
+        Gui::View3DInventorViewer* viewer = view->getViewer();
+
+        // Add 50% to be sure the whole viewport is covered.
+        const float halfAxisLength = 1.5f * viewer->getMaxDimension() / 2.f;
```

Comment:
This is imo needless computation. Be 50 or 75% it doesn't change anything. So : 
`const float halfAxisLength = viewer->getMaxDimension();`